doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -10,7 +10,9 @@
     @subsection qore_08125_new_features New Features in Qore
 
     - added the <a href="../../modules/SalesforceRestClient/html/index.html">SalesforceRestClient</a> module for communicating with Salesforce.com using the REST APIs
-    - module <a href="../../modules/SqlUtil/html/index.html">SqlUtil</a> has support for native default values in tables. See: <a href="https://github.com/qorelanguage/qore/issues/1428">issue 1428</a>
+    - module <a href="../../modules/SqlUtil/html/index.html">SqlUtil</a>
+      - has support for native default values in tables. See: <a href="https://github.com/qorelanguage/qore/issues/1428">issue 1428</a>
```

Comment:
this is a minor cosmetic change, but all the issues in the release notes have the following format:
`- has support for native default values in tables (<a href="https://github.com/qorelanguage/qore/issues/1428">issue 1428</a>)`

please update to look like the other issues so that the release notes are consistent